keras_core/layers/core/input_layer.py

```diff
@@ -45,6 +46,12 @@ def __init__(
         self.batch_shape = tuple(batch_shape)
         self._dtype = backend.standardize_dtype(dtype)
 
+        self.sparse = bool(sparse)
+        if self.sparse and not backend.SUPPORTS_SPARSE_TENSORS:
+            raise ValueError(
+                f"`sparse=True` is not supported with backend: {backend()}"
```

Comment:
`with the '{backend.backend()}' backend`

Consider testing the error message because `backend()` would not have worked

Reply:
Oh, good catch. Added unit test.